build.gradle.kts

```diff
@@ -29,6 +28,13 @@ dependencies {
     testRuntimeOnly(libs.junit.jupiter.engine)
 }
 
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(11))
+        vendor.set(JvmVendorSpec.AZUL)
+    }
+}
```

Comment:
Not sure if this will help but it will at least match the version we're using in GHA build.